Lib/ufo2ft/infoCompiler.py

```diff
@@ -32,9 +30,18 @@ def __init__(self, otf, ufo, info):
         # Create a temporary UFO and sets its fontinfo to the union of the main
         # UFO's fontinfo and the DesignSpace variable-font’s info.
         temp_ufo = type(ufo)()
-        temp_ufo.info = copy.copy(info)
```

Comment:
I think you meant to do `copy.copy(ufo.info)` here, not copy the `info` dict, that's why ufoLib2 was failing to copy.
I don't want to introduce a runtime dep on cattrs just for this.

Reply:
See https://github.com/googlefonts/ufo2ft/pull/816/commits/2020a8dbb21ab9afab4fc8b8fcf34435e6e80db5

Reply:
Silly me! and I was wondering how it worked for me before.